YoCode/PrintToConsole.cs

```diff
@@ -16,6 +16,11 @@ public void PrintIntroduction()
 
         public void PrintFinalResults(TestResults results)
         {
+            this.results = new TestResultFormater(results);
```

Comment:
This doesn't look right, given that line 24 also creates a `TestResultFormatter` but does something different with it.

Reply:
Should be this 
![image](https://user-images.githubusercontent.com/22792365/42462142-45fada82-839a-11e8-9c62-6bb995331e95.png)
For some reason it did not update this